py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -647,11 +647,13 @@ def _msg_query(
             print(response)
         return response
 
-    def reboot(self) -> bool:
+    def reboot(
+        self, purpose: system.RebootRequest.Purpose = system.RebootRequest.Purpose.UPGRADE
+    ) -> bool:
         """TODO: Document"""
```

Comment:
since you're here, replace the doc todo?

Reply:
done